Library/DataSource/UIView-Extensions.swift

```diff
@@ -1,40 +1,38 @@
 import UIKit
 
-extension UIView {
-  public override class func initialize() {
-    struct Static {
-      static var token: dispatch_once_t = 0
-    }
+private func swizzle(_ v: UIView.Type) {
```

Comment:
there's no more `dispatch_once` so this is the dance i have to do for swizzling. 

/cc @stephencelis can you give this a once over?